galasa-managers-parent/galasa-managers-core-parent/dev.galasa.artifact.manager/src/main/java/dev/galasa/artifact/internal/BundleResourcesImpl.java

```diff
@@ -422,6 +422,10 @@ public InputStream zipDirectoryContents(String resourcesDirectory, Map<String, O
     }
 
     private List<String> listDirectory(Bundle bundle, String directory, String fileExtension) {
+    	
+    	if(fileExtension != null && !fileExtension.contains(".")){
```

Comment:
would endsWith be better?

Reply:
I think it should be startsWith as the file extension would be, for instance, either jar or .jar 
But yes that's a good point, I'll change it now.